### PR TITLE
JOE-17: Prototype site footer

### PR DESCRIPTION
**Pull request by martensc**
## Ticket(s)

Please do not submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**

- N/A

**Jira Ticket**

- [JOE-17: Prototype Site Footer](https://palantir.atlassian.net/browse/JOE-17)

## Description

This ticket is for creating the site footer component which will be used for each content type.

## To Test

- [x] Navigate to the [VC Footer component](http://localhost:3000/?p=organisms-vc-footer) and review the markup, scss, and json
- [x] Ensure the component looks good at all viewports
- [x] Change the `theme_mode` value to `light` and ensure the footer background turns dark and the text turns white. (The footer is inverted with the theme mode setting)

## Visual Regressions

N/A

## Relevant Screenshots/GIFs

Screenshot 2023-10-17 at 9 59 04 AM

## Remaining Tasks

N/A

## Additional Notes

N/A

### CODE REVIEW

**Comment by martensc** [comment]
Review and merge this PR once PR #146 has been merged.

**Review by jisung594 — APPROVED** [review]

**Pull request merged**
